VMS/Pages/ActivitySearchPage/AreasPopup.razor.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Blazored.Modal;
+using Microsoft.AspNetCore.Components;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using VMS.Application.Interfaces;
+using VMS.Application.ViewModels;
+
+namespace VMS.Pages.ActivitySearchPage
+{
+    public partial class AreasPopup : ComponentBase
+    {
+        private List<AreaViewModel> areas;
+        private List<int> choosenAreas;
+
+        [Parameter]
+        public List<int> ChoosenAreasList { get; set; }
```

Comment:
For easy reuse, please make this param of type List<AreaViewModel>, i.e this modal will return a list of AreaViewModel instead of that of int.

Reply:
yeah, I have just fixed it and improved some code!